changelogs/unreleased/4994-clayton-gonsalves-minor.md

```diff
@@ -0,0 +1 @@
+Add support for Global External Authorization for HTTPProxy.
```

Comment:
Please add a header to this file, per https://raw.githubusercontent.com/projectcontour/contour/main/changelogs/unreleased/minor-sample.md

it would also be helpful to add a few more sentences explaining the feature, as this will be highlighted in the release notes -- can copy from the guide if appropriate. A link to the guide would also be useful (https://projectcontour.io/docs/v1.25.0/guides/external-authorization/#global-external-authorization).

Some examples of other "minor" changelogs:
https://github.com/projectcontour/contour/blob/main/changelogs/unreleased/5036-relu-minor.md
https://github.com/projectcontour/contour/blob/main/changelogs/unreleased/5080-nsimons-minor.md
https://github.com/projectcontour/contour/blob/main/changelogs/unreleased/5114-fangfpeng-minor.md

Reply:
I added a detailed description here.